opencti-platform/opencti-graphql/config/test.json

```diff
@@ -1,6 +1,8 @@
 {
   "app": {
     "port": 4010,
+    "base_url": "http://localhost:4010/",
```

Comment:
why using localhost here while 127.0.0.1 is used for other existing config like in sync_raw_start_remote_uri ?
should we align all on localhost or either 127.0.0.1 ?

Reply:
all config changes were for the /health as far as I remeber, I will remove.